docs/source/modules/zos_backup_restore.rst

```diff
@@ -48,17 +48,21 @@ data_sets
 
   include
     When *operation=backup*, specifies a list of data sets or data set patterns to include in the backup.
+    When *operation=backup*, specifies a list of data sets or data set patterns to include in the backup.
 
     When *operation=backup* GDS relative names are supported.
 
     When *operation=restore*, specifies a list of data sets or data set patterns to include when restoring from a backup.
 
+    The single asterisk, ``*``, is used in place of exactly one qualifier. In addition, it can be used to indicate to DFSMSdss that only part of a qualifier has been specified.
     The single asterisk, ``*``, is used in place of exactly one qualifier. In addition, it can be used to indicate to DFSMSdss that only part of a qualifier has been specified.
 
+    When used with other qualifiers, the double asterisk, ``**``, indicates either the nonexistence of leading, trailing, or middle qualifiers, or the fact that they play no role in the selection process.
     When used with other qualifiers, the double asterisk, ``**``, indicates either the nonexistence of leading, trailing, or middle qualifiers, or the fact that they play no role in the selection process.
 
     Two asterisks are the maximum permissible in a qualifier. If there are two asterisks in a qualifier, they must be the first and last characters.
 
+    A question mark ``?`` or percent sign ``%`` matches a single character.
     A question mark ``?`` or percent sign ``%`` matches a single character.
```

Comment:
duplicate line

Reply:
regenerated the docs and looks good now

Reply:
I suspect the correction removes a diff so the file does not show up in the PR but is in in the branch as it should be so I am resolving this.
https://raw.githubusercontent.com/ansible-collections/ibm_zos_core/cherry-picked-1110-beta-into-dev/docs/source/modules/zos_backup_restore.rst